source/analysis/SConscript

```diff
@@ -1,6 +1,7 @@
 Import('*')
 
 target = '#output/analysis/plot.eps'       
-source = ['#source/analysis/analysis.do', 
-          '#output/data/data.txt']        
+source = '#source/analysis/analysis.do'
+assets = '#output/data/data.txt'
```

Comment:
Indeed. Please search "Of course we'd need to confirm that we can do this in a way that SCons will treat what's input as dependencies and rebuild if they are modified. I believe we can do this using Depends()." in #19